pandas/tests/series/test_combine_concat.py

```diff
@@ -2,84 +2,27 @@
 import pytest
 
 import pandas as pd
-from pandas import DataFrame, Series
-import pandas._testing as tm
+from pandas import Series
 
 
 class TestSeriesCombine:
-    def test_combine_scalar(self):
-        # GH 21248
-        # Note - combine() with another Series is tested elsewhere because
-        # it is used when testing operators
-        s = pd.Series([i * 10 for i in range(5)])
-        result = s.combine(3, lambda x, y: x + y)
-        expected = pd.Series([i * 10 + 3 for i in range(5)])
-        tm.assert_series_equal(result, expected)
-
-        result = s.combine(22, lambda x, y: min(x, y))
-        expected = pd.Series([min(i * 10, 22) for i in range(5)])
-        tm.assert_series_equal(result, expected)
-
-    def test_update(self):
-        s = Series([1.5, np.nan, 3.0, 4.0, np.nan])
-        s2 = Series([np.nan, 3.5, np.nan, 5.0])
-        s.update(s2)
-
-        expected = Series([1.5, 3.5, 3.0, 5.0, np.nan])
-        tm.assert_series_equal(s, expected)
-
-        # GH 3217
-        df = DataFrame([{"a": 1}, {"a": 3, "b": 2}])
-        df["c"] = np.nan
-
-        df["c"].update(Series(["foo"], index=[0]))
-        expected = DataFrame(
-            [[1, np.nan, "foo"], [3, 2.0, np.nan]], columns=["a", "b", "c"]
-        )
-        tm.assert_frame_equal(df, expected)
-
     @pytest.mark.parametrize(
-        "other, dtype, expected",
-        [
-            # other is int
-            ([61, 63], "int32", pd.Series([10, 61, 12], dtype="int32")),
-            ([61, 63], "int64", pd.Series([10, 61, 12])),
-            ([61, 63], float, pd.Series([10.0, 61.0, 12.0])),
-            ([61, 63], object, pd.Series([10, 61, 12], dtype=object)),
-            # other is float, but can be cast to int
-            ([61.0, 63.0], "int32", pd.Series([10, 61, 12], dtype="int32")),
-            ([61.0, 63.0], "int64", pd.Series([10, 61, 12])),
-            ([61.0, 63.0], float, pd.Series([10.0, 61.0, 12.0])),
-            ([61.0, 63.0], object, pd.Series([10, 61.0, 12], dtype=object)),
-            # others is float, cannot be cast to int
-            ([61.1, 63.1], "int32", pd.Series([10.0, 61.1, 12.0])),
-            ([61.1, 63.1], "int64", pd.Series([10.0, 61.1, 12.0])),
-            ([61.1, 63.1], float, pd.Series([10.0, 61.1, 12.0])),
-            ([61.1, 63.1], object, pd.Series([10, 61.1, 12], dtype=object)),
-            # other is object, cannot be cast
-            ([(61,), (63,)], "int32", pd.Series([10, (61,), 12])),
-            ([(61,), (63,)], "int64", pd.Series([10, (61,), 12])),
-            ([(61,), (63,)], float, pd.Series([10.0, (61,), 12.0])),
-            ([(61,), (63,)], object, pd.Series([10, (61,), 12])),
-        ],
+        "dtype", ["float64", "int8", "uint8", "bool", "m8[ns]", "M8[ns]"]
     )
-    def test_update_dtypes(self, other, dtype, expected):
+    def test_concat_empty_series_dtypes_match_roundtrips(self, dtype):
```

Comment:
this change looks good but better if not done at the same time as a move (in future).  could any of the dtype fixtures in conftest be used here instead? (as a follow-up)